evap/evaluation/tools.py

```diff
@@ -115,7 +117,7 @@ def translate(**kwargs):
     return property(lambda self: getattr(self, kwargs[get_language() or "en"]))
 
 
-def clean_email(email):
+def clean_email(email: str) -> str:
```

Comment:
This is actually `str | None`, see for example `UserForm.clean_email`. Not sure if we can model a `None` iff `None` relationship with generics or so, would be cool